.github/workflows/r.yml

```diff
@@ -177,6 +177,9 @@ jobs:
         - { rtools: 40, arch: 'mingw32' }
         - { rtools: 40, arch: 'mingw64' }
         - { rtools: 40, arch: 'ucrt64' }
+        - { rtools: 42, arch: 'mingw32' }
+        - { rtools: 42, arch: 'mingw64' }
+        - { rtools: 42, arch: 'ucrt64' }
```

Comment:
I don't think you need these. The C++ libraries built with rtools40 will work with R with rtools42.

Reply:
I removed them.